README.md

```diff
@@ -1,8 +1,97 @@
-# hackshare
+# HackShare
 
-### How to Use
+A web app where fellows can teach other fellows the skills they believe they are good at.
```

Comment:
I think share would be better to use instead of teach i.e. if we are going with tagline "Share and Learn"